core/controllers/base_test.py

```diff
@@ -1025,6 +1025,11 @@ def test_every_method_has_decorator(self):
                 handlers_checked.append(
                     (handler.__name__, 'DELETE', handler_is_decorated))
 
+            if handler.head != base.BaseHandler.head:
+                handler_is_decorated = hasattr(handler.delete, '__wrapped__')
+                handlers_checked.append(
+                    (handler.__name__, 'HEAD', handler_is_decorated))
```

Comment:
What does this do?

Reply:
test_every_method_has_decorator had all the request handlers so I added head in that as well.

Reply:
But I don'ŧtthink we expect `head` function in any handlers. HEAD should just be GET but with only headers returned